core/src/main/java/com/graphhopper/routing/ev/NewCountry.java

```diff
@@ -0,0 +1,265 @@
+    IRL("Ireland"),
+    IRN("Iran"),
+    IRQ("Iraq"),
+    ISL("Iceland"),
+    ISR("Israel"),
+    ITA("Italy"),
+    JAM("Jamaica"),
+    JEY("Jersey"),
+    JOR("Jordan"),
+    JPN("Japan"),
+    KAZ("Kazakhstan"),
+    KEN("Kenya"),
+    KGZ("Kyrgyzstan"),
+    KHM("Cambodia"),
+    KIR("Kiribati"),
+    KNA("Saint Kitts and Nevis"),
+    KOR("South Korea"),
+    KWT("Kuwait"),
+    LAO("Laos"),
+    LBN("Lebanon"),
+    LBR("Liberia"),
+    LBY("Libya"),
+    LCA("Saint Lucia"),
+    LIE("Liechtenstein"),
+    LKA("Sri Lanka"),
+    LSO("Lesotho"),
+    LTU("Lithuania"),
+    LUX("Luxembourg"),
+    LVA("Latvia"),
+    MAR("Morocco"),
+    MCO("Monaco"),
+    MDA("Moldova"),
+    MDG("Madagascar"),
+    MDV("Maldives"),
+    MEX("Mexico"),
+    MHL("Marshall Islands"),
+    MKD("North Macedonia"),
+    MLI("Mali"),
+    MLT("Malta"),
+    MMR("Myanmar"),
+    MNE("Montenegro"),
+    MNG("Mongolia"),
+    MOZ("Mozambique"),
+    MRT("Mauritania"),
+    MSR("Montserrat"),
+    MUS("Mauritius"),
+    MWI("Malawi"),
+    MYS("Malaysia"),
+    NAM("Namibia"),
+    NER("Niger"),
+    NGA("Nigeria"),
+    NIC("Nicaragua"),
+    NIU("Niue"),
+    NLD("Netherlands"),
+    NOR("Norway"),
+    NPL("Nepal"),
+    NRU("Nauru"),
+    NZL("New Zealand"),
+    OMN("Oman"),
+    PAK("Pakistan"),
+    PAN("Panama"),
+    PCN("Pitcairn Islands"),
+    PER("Peru"),
+    PHL("Philippines"),
+    PLW("Palau"),
+    PNG("Papua New Guinea"),
+    POL("Poland"),
+    PRK("North Korea"),
+    PRT("Portugal"),
+    PRY("Paraguay"),
+    PSE("Palestinian Territories"),
+    QAT("Qatar"),
+    ROU("Romania"),
+    RUS("Russia"),
+    RWA("Rwanda"),
+    SAU("Saudi Arabia"),
+    SDN("Sudan"),
+    SEN("Senegal"),
+    SGP("Singapore"),
+    SGS("South Georgia and the South Sandwich Islands"),
+    SHN("Saint Helena, Ascension and Tristan da Cunha"),
+    SLB("Solomon Islands"),
+    SLE("Sierra Leone"),
+    SLV("El Salvador"),
+    SMR("San Marino"),
+    SOM("Somalia"),
+    SRB("Serbia"),
+    SSD("South Sudan"),
+    STP("São Tomé and Príncipe"),
+    SUR("Suriname"),
+    SVK("Slovakia"),
+    SVN("Slovenia"),
+    SWE("Sweden"),
+    SWZ("Eswatini"),
+    SYC("Seychelles"),
+    SYR("Syria"),
+    TCA("Turks and Caicos Islands"),
+    TCD("Chad"),
+    TGO("Togo"),
+    THA("Thailand"),
+    TJK("Tajikistan"),
+    TKL("Tokelau"),
+    TKM("Turkmenistan"),
+    TLS("East Timor"),
+    TON("Tonga"),
+    TTO("Trinidad and Tobago"),
+    TUN("Tunisia"),
+    TUR("Turkey"),
+    TUV("Tuvalu"),
+    TWN("Taiwan"),
+    TZA("Tanzania"),
+    UGA("Uganda"),
+    UKR("Ukraine"),
+    URY("Uruguay"),
+    USA("United States"),
+    UZB("Uzbekistan"),
+    VAT("Vatican City"),
+    VCT("Saint Vincent and the Grenadines"),
+    VEN("Venezuela"),
+    VGB("British Virgin Islands"),
+    VNM("Vietnam"),
+    VUT("Vanuatu"),
+    WSM("Samoa"),
+    XKX("Kosovo"),
+    YEM("Yemen"),
+    ZAF("South Africa"),
+    ZMB("Zambia"),
+    ZWE("Zimbabwe"),
+    MISSING("missing");
```

Comment:
Can we move this one to the top? The other EV enums use the first entry to encode the most permissive or indicate a missing/unknown value.

Reply:
Ok done.